.travis.yml

```diff
@@ -5,6 +5,10 @@ addons:
       - deadsnakes
     packages:
       - python3.6
+      - python3.7
+      - python3.7-distutils
+      - python3.8
+      - python3.8-distutils
```

Comment:
why do we need to add `python3.8-distutils` for python 3.8 and 3.7, but not for 3.6

Reply:
There is no `python3.6-distutils` package - for some reason it seems to have `distutils` included.  See related travis CI job: https://travis-ci.org/Shopify/ci-queue/jobs/609006985

However, without `distutils` installed in the standard library, `tox` chokes up https://travis-ci.org/Shopify/ci-queue/jobs/609003518